flink-core/src/test/resources/log4j-test.properties

```diff
@@ -18,7 +18,7 @@
 
 # Set root logger level to OFF to not flood build logs
 # set manually to INFO for debugging purposes
-log4j.rootLogger=OFF, testlogger
+log4j.rootLogger=ERROR, testlogger
```

Comment:
We can. I actually found this useful for debugging because you generally want to see ERRORs :) Can be resolved in a different issue though.